mRemoteV1/Resources/Help/Introduction.htm

```diff
@@ -1,45 +1,50 @@
-        View remote session info and log off sessions (RDP)<br />
-        Portable (with the use of command line switches)<br />
-        &quot;Auto-Update&quot; feature<br />
-        Show description tooltips when hoovering over connections<br />
-        System tray icon with connection menu<br />
-        Fullscreen (Kiosk) mode<br />
-        Assign global credentials to use when no information is provided on connection basis<br />
-        Host Up/Down (Ping) feature shows if the selected host answers to a ping</p>
-    <p class="heading1">
-        License</p>
-    <p>
-        The application is released under the <a href="../License.txt">GPL</a> (V2)<br />
-        Binary and source code packages are available for download.</p>
+  <h1>Introduction</h1>
+  <hr>
+  <p>
+    mRemoteNG is a multi-protocol remote connections manager.
+    <br>
+    As of Version 1.00 it supports the RDP, VNC, ICA, SSH, Telnet, RAW, Rlogin and HTTP/S protocols.
+    <br>
+    The main goal is to minimize window clutter and to provide a easy gui for managing all your remote connections.
+  </p>
+  <h1>Features</h1>
+  <hr>
+  <ul>
+    <li>Free and Open Source, released under the GPL</li>
+    <li>Panels and tabs allow to group certain connections together, dock them to any side of the window or completely undock them and move them to another screen for example</li>
+    <li>Multiple supported protocols (RDP, VNC, ICA, SSH, Telnet, RAW, Rlogin and HTTP/S)</li>
+    <li>Easy to organize and maintain list of connections</li>
+    <li>Inheritance makes it possible to store properties on folder basis and let the underlying connections inherit this info</li>
+    <li>Support for importing connections from Active Directory</li>
+    <li>Allows creating nested containers (folders) to categorize connections</li>
+    <li>&quot;Quick Connect&quot; feature to quickly open a connection without creating an entry</li>
+    <li>&quot;Quick Search&quot; feature to quickly find a connection while typing</li>
+    <li>Support for SCP/SFTP (SSH) file transfers</li>
+    <li>Assign icons to connections to easily identify purpose</li>
+    <li>Screenshot manager allows to collect multiple screenshots and save them all together or copy them to the clipboard</li>
+    <li>View remote session info and log off sessions (RDP)</li>
+    <li>Portable (with the use of command line switches)</li>
+    <li>&quot;Auto-Update&quot; feature</li>
+    <li>Show description tooltips when hoovering over connections</li>
+    <li>System tray icon with connection menu</li>
+    <li>Fullscreen (Kiosk) mode</li>
+    <li>Assign global credentials to use when no information is provided on connection basis</li>
+    <li>Host Up/Down (Ping) feature shows if the selected host answers to a ping</li>
+  </ul>
+  <h1>License</h1>
+  <hr>
+  <p>
+    The application is released under the <a href="../License.txt">GPL</a> (V2)
```

Comment:
looks like its named: License.txt inside the installed release of mRemoteNG so on the computer it is placed in that path it seems. On the other hand.....it could just as well be placed inside Help folder that way it can be copied.

Reply:
Thats a good point, i don't think we do some of the file moves in local builds that are done during the release builds. That may be something we should change.